server/sonar-web/src/main/js/apps/projectsManagement/__tests__/ProjectRow-test.tsx

```diff
@@ -44,17 +43,11 @@ it('checks project', () => {
   expect(onProjectCheck).toBeCalledWith(project, false);
 });
 
-it('applies permission template', () => {
```

Comment:
Could be nice to create a test for `ProjectRowActions` and add that test back there.